arm-relay/2017-04-01/swagger/relay.json

```diff
@@ -1575,9 +1581,6 @@
           "description": "Resource tags"
         }
       },
-      "required": [
```

Comment:
Should the models that inherit from here be Tracked Resources? I'm asking because by removing the "required" location, they are not considered "tracked" - see definition of tracked resource at https://github.com/Azure/azure-rest-api-specs-pr/blob/master/common-types/resource-management/v1/types.json#L23